.github/workflows/build.yml

```diff
@@ -15,70 +15,16 @@ jobs:
         uses: MarceloPrado/has-changed-path@v1.0
         id: changed
         with:
-          paths: urban lib
+          paths: src lib
```

Comment:
Could probably remove this conditional since it's not very accurate at detecting vehicle-specific changes now.

It's not a huge deal since builds run in parallel and the minutes are free for public repos.

Reply:
thanks, done!